demisto_sdk/commands/format/update_layout.py

```diff
@@ -267,6 +267,10 @@ def remove_non_existent_fields_container_layout(self):
         Remove non-existent fields from a container layout.
         """
         if not self.id_set_file:
+            click.secho(
```

Comment:
please replace with `logger = logging.getLogger('demisto-sdk')`, and `logger.warning`